Assets/HoloToolkit/Input/Scripts/GestureManager.cs

```diff
@@ -258,8 +264,17 @@ void LateUpdate()
 
             bool focusedChanged = FocusedObject != newFocusedObject;
 
+            if (focusedChanged)
+            {
+                // If the currently focused object doesn't match the new focused object, cancel the current gesture.
+                // Start looking for new gestures.  This is to prevent applying gestures from one hologram to another.
+                gestureRecognizer.CancelGestures();
+                FocusedObject = newFocusedObject;
+                gestureRecognizer.StartCapturingGestures();
+            }
+
 #if UNITY_EDITOR
-            if (Input.GetMouseButtonDown(1) || Input.GetKeyDown(EditorSelectKey))
+            if ((Input.GetMouseButtonDown(1) || Input.GetKeyDown(EditorSelectKey)) && !focusedChanged)
```

Comment:
If I understand this correctly, you will only accept a tap if I looked away and saw the object again? What if I wanted to tap on the same object for some other interaction state to show up?